optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -222,9 +222,10 @@ def _open_browser(html_path):
         setsid = getattr(os, 'setsid', None)
         if not setsid:
             setsid = getattr(os, 'setpgrp', None)
+        inout = file(os.devnull, "r+")
         cmd = ['xdg-open', html_path]
-        subprocess.Popen(cmd, close_fds=True, stdin=subprocess.DEVNULL,
-                         stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL,
+        subprocess.Popen(cmd, close_fds=True, stdin=inout,
+                         stdout=inout, stderr=inout,
```

Comment:
yes, my bad. I assumed I was reading Py2 documentation but I actually read the Py3 one, twice.